src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+const items = document.querySelectorAll('li');
+const arr = [...items];
```

Comment:
arr - is not the best name for a variable, it would be better if its name reflected what is inside it.